firefox_puppeteer/base.py

```diff
@@ -2,10 +2,11 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
+import errors
+
 
 class BaseLib(object):
-    """A trivial base class that handles lazily setting the "client" class
-    attribute."""
+    """A base class that handles lazily setting the "client" class attribute."""
 
```

Comment:
The line length is not a problem given that we allow 100 chars now. Also I updated it to keep it in sync with the `UIBaseLib` class, which I have added below. So it's not totally unrelated.